### PR TITLE
Projects edit

### PR DESCRIPTION
**Pull request by javierforero**
@droberts84 This is what I got so far. What do you think about the projects section? I put the source code button at the bottom. I am still working in it, but I kinda need a second eye

### CODE REVIEW

**Comment by drobtravels** [comment]
Make sure you add a `README` and are using bundler.  (IE I should be able to type `bundle install`, instead of having to know to `gem install jekyll` or any other gems you are using).

The button at the bottom for source code is fine.  As far as the specific text, I want to think about it more and will get back to you.

File: _posts/2016-08-04-challenges-in-developing.md [hunk 1]
```diff
@@ -4,4 +4,16 @@ date: 2016-08-04
...
+
+“Alexa, play The Beatles on Spotify”.
+
+And if you do have a working Spotify account, Alexa will hook you up with The Beatles. Behind the scenes, though, your words are being converted into JSON and are sent as a request to a server (most likely AWS), which contains the business logic to make the song play from Spotify.
+
+My first idea for a skill was simple. I wanted to build a slang dictionary where the user would ask Alexa for the slang definition of a word and Alexa would get the Urban Dictionary definition and read it to the user.
+
+I liked the idea of this skill because to me it was funny, and it required me to work with UrbanDictionary's APIs and use asynchronous programming in order to make it work. The most challenging thing about this project was finding documentation and understanding asynchronous programming. In the end, my skill was not published because of its indecent nature, but it was a great experience over all.
```
Comment (drobtravels) [inline _posts/2016-08-04-challenges-in-developing.md:19]: but you did publish other skills!

File: _projects/blocmetrics.md [hunk 2]
```diff
@@ -1,33 +1,16 @@
...
-
-What worked really well for Blocmetrics was having two charts. A pie chart that displays the frequency of an event and a line chart that displays when events were created. Although this is very useful information, most users want to be able to choose how they want to see the data and also have a way to export it as a csv file. These are definitely improvements that will be made available.
-
-![alt text](../img/blocmetrics.png)
-

-**Conclusion**
-
-Blocmetrics is a powerful application that is easy to integrate with any web application.
+This web application was made to help users track events such as clicks or impressions on their websites. Blocmetrics allows users to track as many events from their websites and displays them graphically providing instant insights.
```
Comment (drobtravels) [inline _projects/blocmetrics.md:16]: do you want to mention anything about challenges associated with accepting clientside API requests from other applications?

Comment (javierforero) [reply _projects/blocmetrics.md:16]: Like the limitations that app has or the challenges I had getting it to work?

Comment (drobtravels) [reply _projects/blocmetrics.md:16]: dealing with CORS

Comment (javierforero) [reply _projects/blocmetrics.md:16]: gotcha so explain what I had to do get around CORS?

Comment (drobtravels) [reply _projects/blocmetrics.md:16]: yup

File: _posts/2016-08-04-challenges-in-developing.md (hunk 1 above)
Comment (javierforero) [reply _posts/2016-08-04-challenges-in-developing.md:19]: I did. I will mention it here at the end

**Pull request merged**